Automattic/wp-calypso
# Paid stats: restrict date picker to 7 days instead of 30

Fixes https://github.com/Automattic/jetpack-roadmap/issues/2017

## Proposed Changes

* Limit date picker access to 7 days unless the user has paid stats

Todo:
- [ ] Fix the styling of the disabled date ranges
- [ ] Ensure the 7 day shortcut is selected

## Testing Instructions

* Load stats for a site with no plan
* Confirm the date picker is defaults to and is limited to 7 days

Before

![Screenshot(152)](https://github.com/user-attachments/assets/6f720449-f4ab-467f-9df0-329edf649f28)

After

![Screenshot(153)](https://github.com/user-attachments/assets/ceaed27f-0c22-416e-91b3-c6980e0c7f59)
